tailwind.config.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
 
 module.exports = {
-  content: ["./src/**/*.{html,js,ts,tsx}"],
+  content: ["./src/**/*.{html,js,ts,tsx}", "./documentation/**/*.tsx"],
```

Comment:
I vaguely remember at one of our discussions that we might potentially want to move off of using tailwind altogether for ui-seeds. I initially brought it in because a lot of our styling was tied to it, but don't know how necessary it is. Curious on your thoughts around that? @emilyjablonski do you have thoughts?

Reply:
Admittedly I'm not big on Tailwind these days, but I think we'd need to talk about how we might support some of the one-off page layout sort of tasks that Tailwind makes easier…alternatives could be we develop a set of most common utility classes ourselves directly based off design tokens ([prior art here](https://nordhealth.design/css/#installation), for example), or we produce some sort of guidance on how individual pages/layouts could create stylesheets that are scoped in some way as not to interfere with other pages. I've also heard of some tools out there which can autogenerate utility classes based on a design system config but is less "overhead"/opinionated compared to Tailwind. I'm also curious other folks thoughts here.

Reply:
@jaredcwhite I think that conversation makes sense for core, but I think we could probably move forward without those concerns in ui-seeds removing Tailwind?